src/rules/completedDocsRule.ts

```diff
@@ -307,7 +307,22 @@ const modifierAliases: { [i: string]: string } = {
     export: "exported",
 };
 
-function walk(context: Lint.WalkContext<ExclusionsMap>, typeChecker: ts.TypeChecker) {
+/**
+ * @remarks See https://github.com/ajafff/tsutils/issues/16
+ */
+const getApparentJsDoc = (node: ts.Node): ts.JSDoc[] => {
+    if (node.kind === ts.SyntaxKind.VariableDeclaration) {
+        const parent = node.parent as ts.VariableDeclarationList;
```

Comment:
this assumtion is not always true... well, it might be in the context of this rule

Reply:
I think it is for this rule. Just to be safe, I kept the function internal to this file instead of making it more globally available (pending tsutils having its own).